src/truststore/__init__.py

```diff
@@ -1,13 +1,34 @@
 """Verify certificates using native system trust stores"""
 
+import ssl
 import sys as _sys
 
 if _sys.version_info < (3, 10):
     raise ImportError("truststore requires Python 3.10 or later")
 
+# Detect Python runtimes which don't implement SSLObject.get_unverified_chain() API
+# This API only became public in CPython 3.13 but was available in CPython since 3.10.
```

Comment:
Could we do this check for Python < 3.13 only, to avoid unnecessary processing at import time?